source/door_b_enb.rst

```diff
@@ -0,0 +1,174 @@
+.. _dpaa-shutdown:
+
+ACIS Door Mechanism side B Anomalous Enable
+===========================================
+
+What is it?
+-----------
+
+The ACIS Door Mechanism side B was enabled anomalously, presumably due to a spurious command.
+
+When did it happen before?
+--------------------------
+
+The ACIS Door Mechanism side B has been enabled anomalously only once during the mission:
+
+* March 20, 2018: 2018:079:23:42:10, obsid 20135
+
+This page also lists symptoms and responses for a side A anomaly, which has not happened in flight.
+
+Will it happen again?
+---------------------
+
+It appears likely that this anomaly, or similar ones, will occur again if the mission continues.
+
+How is this anomaly diagnosed?
+------------------------------
+
+Within a major frame (32.2 seconds), one should see:
+
+* 1MDBUBON (Door Enable side B) change from 0 to 1 (Disabled to Enabled). Side A: 1MDBUAON.
+* 1MCDRBCL (Door close drive side B) is OFF. Side A: 1MCDRACL.
+* Temperatures 1MAHOBT and 1MAHCBT will show credible values (-2 C in this case), due
+  to the thermistors being powered up when the mechanism is enabled. Side A: 1MAHOAT and 1MAHCAT.
+* 1DACTAT (the Door Angle Potentiometer) should be unchanged at 70 +/- 5 degrees,
+  indicating the door has not moved.
+* ACIS should still be recieving photons if a science run is in progress.
```

Comment:
We might want to mention here that for really faint sources the count rate will not be that much above what we expect for the instrumental background if the door was closed (correct?)

Reply:
Good point.